wrapper/log_parser.py

```diff
@@ -13,7 +13,7 @@ class OutputParser(object):
     COPY_DISK_RE = re.compile(br'.*Copying disk (\d+)/(\d+) to.*')
     DISK_PROGRESS_RE = re.compile(br'\s+\((\d+\.\d+)/100%\)')
     NBDKIT_DISK_PATH_RE = re.compile(
-        br'nbdkit: debug: Opening file (.*) \(.*\)')
+        br'nbdkit: .*: debug: Opening file (.*) \(.*\)')
```

Comment:
I you would just do
```suggestion
        br'nbdkit:.* debug: Opening file (.*) \(.*\)')
```
then it would work with both the new version of virt-v2v and the old one as well. But that's just a nitpick.